lib/cron.js

```diff
@@ -406,6 +406,10 @@ module.exports = function(configure) {
 
 			}
 
+			// Safey check to prevent setting checkpoint to undefined
```

Comment:
Keeps the existing checkpoint in place if don't send one.  This is a product of using partial_start but if the lambda were to die between these checkpoints and the real one it would start the bot over from the start of the day